console_backend/src/ntrip_tab.rs

```diff
@@ -245,27 +258,29 @@ fn main(
 
     let progress = Arc::new(Mutex::new(Progress::default()));
     let progress_clone = progress.clone();
-    let running_clone = is_running.clone();
-    let progress_thd = thread::spawn(move || loop {
-        {
-            if !*running_clone.lock().unwrap() {
-                break;
+    let progress_thd = thread::spawn({
+        let running = is_running.clone();
+        move || loop {
+            {
+                if !running.get() {
+                    return false;
+                }
             }
+            {
+                let mut progress = progress_clone.lock().unwrap();
+                heartbeat.set_ntrip_ul(progress.tick_ul());
+                heartbeat.set_ntrip_dl(progress.tick_dl());
```

Comment:
This only shows NTRIP client data (sent and received in RTCM).
Does not show converted RTCM to SBP size that is being sent to device.